src/pages/index.tsx

```diff
@@ -73,13 +73,13 @@ const IndexPage = ({ PSEData: data, blockData, trustPositifData }: IndexPageProp
       <div className='flex flex-col bg-brand-bg-light'>
         <SiteHeader />
         <PageContent>
-          <div className='mx-auto w-full max-w-screen-sm'>
+          <div className='mx-auto w-full'>
```

Comment:
```suggestion
          <div className='mx-auto w-fullmax-w-screen-xl'>
```

Idem.

Reply:
Chiming in, old version of the website uses `max-w-screen-xl`
https://github.com/angeloanan/kominfo-d-/blob/5a958fbbd4aacb1fc41c78d88b5cb42034b04a61/src/pages/index.tsx#L71